releasenotes/notes/fix-chained-cni-helm.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: release-notes/v2
+kind: feature
+area: installation
+issue:
+  - 43632
+  - 45034
+releaseNotes:
+- |
+  **Fixed** parity with helm charts not including `istio_cni` similar to `istioctl`.
```

Comment:
This is probably fine and overly pedantic, but nothing except documentation was _fixed_; this implies its a bug fix.